samples/uri/web/GreatNotes/StoreApp/AppxManifest.EdgeCanary.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" xmlns:uap3="http://schemas.microsoft.com/appx/manifest/uap/windows10/3" xmlns:uap10="http://schemas.microsoft.com/appx/manifest/uap/windows10/10" xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities" xmlns:rescap3="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities/3" xmlns:desktop2="http://schemas.microsoft.com/appx/manifest/desktop/windows10/2" IgnorableNamespaces="uap uap3 rescap desktop2">
+  <Identity Name="net.azurestaticapps.6.ambitious-flower-0ec6e8610.e" Publisher="CN=3a54a224-05dd-42aa-85bd-3f3c1478fdca, OID.2.25.311729368913984317654407730594956997722=1" Version="1.0.1.0" ProcessorArchitecture="neutral"/>
+  <Properties>
+    <DisplayName>Greatnotes</DisplayName>
+    <PublisherDisplayName>Contoso, Inc.</PublisherDisplayName>

```

Comment:
should we make msft the publisher for both apps?

Reply:
fixed